docs/topics/sequences.md

```diff
@@ -177,6 +177,7 @@ When the result size reaches 4, the processing stops because it's the largest po
 
 The sequence processing goes like this:
 
-![Sequences processing](sequence-processing.png) {width="700"}
+![Sequences processing](sequence-processing.svg) {width="700"}
 
-In this example, the sequence processing takes 18 steps instead of 23 steps for doing the same with lists.
+In this example, by processing elements lazily and stopping once four items are found, sequences reduce the total number
```

Comment:
Would something simpler work? Like:
 In this example, lazy processing of elements and stopping after finding four items minimizes the number of operations compared to a list approach.